notebooks/text_models/solutions/load_text.ipynb

```diff
@@ -0,0 +1,1935 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            print("Label:", label_batch.numpy()[i])
</pre><p>I would add s separater after this like <code>print("*" * 50)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>+1 (the output is a bit hard to read)</p>

Reply:
<p>Updated the code:</p><ul><li>Converted the Question which was in bytes type to a string to make it more readable </li><li>Added a separator between each instance</li></ul>